frontend/src/components/VHeader/VFilterIconOrCounter.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <VIcon v-if="showIcon" :icon-path="filterIcon" />
+  <p
+    v-else
+    class="flex h-6 w-6 items-center justify-center"
```

Comment:
The counter needs to have a corner radius of `2px`. I think the class is `rounded-sm`.